File: My_java_kotlindsl_uniappx_app/app/src/main/java/com/example/my_java_kotlindsl_uniappx_app/MyApplication.java
package com.example.my_java_kotlindsl_uniappx_app;
//các thư viện cần khai báo
import android.app.Application;
import io.dcloud.feature.sdk.DCSDKInitConfig;
import io.dcloud.feature.sdk.DCUniMPSDK;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        //Cấu hình SDK
        DCSDKInitConfig config = new DCSDKInitConfig.Builder().build();
        // Khởi tạo SDK
        DCUniMPSDK.getInstance().initialize(this, config);
    }
}

File: My_kotlin_groovydsl_uniappx_app/app/src/main/java/com/example/my_kotlin_groovydsl_uniappx_app/MyApplication.kt
package com.example.my_kotlin_groovydsl_uniappx_app
//các thư viện cần khai báo
import android.app.Application
import io.dcloud.feature.sdk.DCSDKInitConfig
import io.dcloud.feature.sdk.DCUniMPSDK

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        // Cấu hình SDK
        val config = DCSDKInitConfig.Builder().build()

        // Khởi tạo SDK
        DCUniMPSDK.getInstance().initialize(this, config)
    }
}